src/main/kotlin/ink/abb/pogo/scraper/tasks/UpdateProfile.kt

```diff
@@ -111,5 +117,26 @@ class UpdateProfile : Task {
         } catch (e: Exception) {
             Log.red("Failed to update profile and inventories")
         }
+        if(ctx.api.playerProfile.playerData.team==TeamColor.NEUTRAL && ctx.api.playerProfile.stats.level>=5) {
+            if(settings.autoTeam.toUpperCase().equals("RED") || settings.autoTeam.toUpperCase().equals("BLUE") || settings.autoTeam.toUpperCase().equals("YELLOW")) {
+	        	val msg = SetPlayerTeamMessage.newBuilder().setTeam(TeamColor.valueOf(settings.autoTeam.toUpperCase())).build()
```

Comment:
Little stylecheck: can you replace the tabs with spaces, put a space between the `if<->(`'s and remove the `;` at the end of some lines?